backend/app/src/main/java/com/moh/phlat/backend/esb/json/PlrResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.moh.phlat.backend.esb.json;
+
+import java.text.SimpleDateFormat;
+
+import com.moh.phlat.backend.esb.boundary.PlrToken;
+import com.moh.phlat.backend.model.ProcessData;
+
+public interface PlrResponse {
```

Comment:
PlrResponse is too generic name. we need to use some functionality specific name. An umbrella term for facility+hds+relation. Are we going to upload any other data?

Reply:
I've been able to utilize PlrResponse to share commonly used Custom JSON node names and common PLR response codes with the specific response classes. Also, since we'll be sending Validation requests to PLR, this PlrResponse interface will give a good baseline to help with developing those requests, and down the road there could be more types of requests to send, such as other relationship types or even other provider types.